Need help testing the Linux build

I don't really have a working Linux setup right now. Can someone verify that the project builds on Linux? I would like to push a new version out with minor fixes.

You will have to forgive me, as this might be something simple, but I just don't work with C++ often (at all), but I attempted to build and got the following:

```log
CMake Error at CMakeLists.txt:7 (project):
  Generator
    Unix Makefiles
  does not support platform specification, but platform
    x64
  was specified.
-- Configuring incomplete, errors occurred!
See also "/mnt/x/GitHub/kmammou/v-hacd/app/CMakeFiles/CMakeOutput.log".
```

<details>
<summary>System Details</summary>

mosthated@pop-os
----------------
OS: Pop!_OS 22.04 LTS x86_64
Host: X670E Taichi
Kernel: 6.0.2-060002-generic
Uptime: 1 day, 16 hours, 20 mins
Packages: 4355 (dpkg), 43 (nix-default), 99 (flatpak), 14 (snap)
Shell: zsh 5.8.1
Resolution: 2560x1080
DE: GNOME 42.3.1
WM: Mutter
WM Theme: Yaru-dark
Theme: Pop-dark [GTK2/3]
Icons: Flatery-Orange-Dark [GTK2/3]
Terminal: alacritty
CPU: AMD Ryzen 9 7950X (32) @ 5.881GHz
GPU: AMD ATI Radeon RX 6800/6800 XT / 6900 XT
GPU: AMD ATI 59:00.0 Device 164e
Memory: 42380MiB / 63415MiB

</details>

--- a/README.md
+++ b/README.md
@@ -15,7 +15,13 @@ A second approach consists in computing an exact convex decomposition of a surfa
 # Installing the Package
 
 On Windows, go to the ./app directory and run 'cmake -DCMAKE_GENERATOR_PLATFORM=x64 CMakeLists.txt' and then load the solution file
-On Linux, go to the ./app directory and run 'cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_GENERATOR_PLATFORM=x64 CMakeLists.txt' and then run 'cmake --build .' to build the test app
+
+On Linux, use this:
+```bash
+cd app
+cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
+cmake --build build
+```
 
 # Documentation
 Click this link to find detailed documentation for how to use the library:

--- a/app/SaveUSDA.cpp
+++ b/app/SaveUSDA.cpp
@@ -3,6 +3,7 @@
 #include <float.h>
 #include <math.h>
 #include <vector>
+#include <cstdio>
 
 #ifdef _MSC_VER
 #pragma warning(disable:4100 4996)

